tester/Makefile

```diff
@@ -16,7 +16,7 @@ setup:
 	GO15VENDOREXPERIMENT=1 glide install
 
 build: *.go fmt
-	go build -o $(BINARY_NAME) $(GOBUILD_VERSION_ARGS) github.com/jtblin/$(BINARY_NAME)
+	go build -o $(BINARY_NAME) $(GOBUILD_VERSION_ARGS) github.com/jtblin/gostatsd/tester
```

Comment:
This was not working because package name is not the same as binary name. Am I doing it wrong?